tox.ini

```diff
@@ -7,8 +7,8 @@
 envlist =
     fmt
     lint
+    pyright
     py{312,311,310,39,38,37,36,py3}
-skip_missing_interpreters = true
```

Comment:
Do not rely on skipping missing interpreters, always run a particular version and fail if interpreter is not available.